chart/templates/injector_role.yaml

```diff
@@ -14,6 +14,8 @@ rules:
   - pods
   verbs:
   - list
+  - get
+  - update
```

Comment:
Just to confirm, we need `get` and `update` in order to find the pod and update it once we remove its finalizer correct?

Reply:
Exactly, yes.